src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1119,12 +1087,8 @@ def execute(self, ppl):
             part_name="plant_gen",
             cols_to_keep=["plant_part"],
             one_to_many=True,
-        )[["record_id_eia_og", "record_id_eia", "plant_part_og"]].rename(
-            columns={
-                "record_id_eia": "gen_id",
-                "record_id_eia_og": "record_id_eia",
-                "plant_part_og": "plant_part",
-            },
+        )[["record_id_eia", "record_id_eia_plant_gen", "plant_part"]].rename(
+            columns={"record_id_eia_plant_gen": "gen_id"},
```

Comment:
this is another instance of a change in which side of the `match_to_single_plant_part` merge gets a suffix